lib/push.js

```diff
@@ -49,49 +61,76 @@ module.exports = function (log, db) {
      * @promise
      */
     notifyUpdate: function notifyUpdate(uid, reason) {
-      var events = reasonToEvents[reason] || reasonToEvents.accountVerify
+      reason = reason || 'accountVerify'
+      return this.pushToDevices(uid, reason)
+    },
+
+    notifyNewDevice: function notifyNewDevice(uid, deviceName, curDeviceId) {
+      var data = new Buffer(JSON.stringify({
+        action: 'fxacounts:new-device',
+        data: {
+          deviceName: deviceName
+        }
```

Comment:
Nice and simple! It looks a lot like the WebChannel format, except `command` is spelled `action` here. I think adding a `version` field would be good, too, per Richard's suggestion on the mailing list.